### PR TITLE
JetBrains Editor doc updates

### PR DESCRIPTION
**Pull request by filiptronicek**
Update JetBrains docs and remove the private beta page.

Fixes https://github.com/gitpod-io/gitpod/issues/6576

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1315"></a>

### CODE REVIEW

File: src/routes/docs/editors/goland.md [hunk 1]
```diff
@@ -11,16 +11,32 @@ title: GoLand
...
+JetBrains GoLand is currently in Beta. We have a [JetBrains channel on our Discord (#jetbrains-beta)](https://www.gitpod.io/chat). Feel free to discuss any issues / leave feedback there. If you prefer, you can also leave feedback on the dedicated [GitHub Issue](https://github.com/gitpod-io/gitpod/issues/6576).
 
 > **Warning:** Currently, our JetBrains implementation uses [Code With Me](https://www.jetbrains.com/code-with-me/) links and a shared password to facilitate early Beta testing. This means that for now, anyone who might discover the workspace URL could potentially get full access to your workspace. This solution is temporary and will be resolved before the final release.
 
+### Starting all workspaces in GoLand by default
+
+1. Go to [your preferences page](https://gitpod.io/preferences)
+2. Select "Open In Desktop IDE" and choose <kbd>GoLand</kbd>.
+3. Start a _new workspace_ and you will be prompted on workspace start to use JetBrains or VS Code Web
```
Comment (akosyakov) [inline src/routes/docs/editors/goland.md:22]: I think `or VS Code Web` is not necessary to mention here.

File: src/routes/docs/editors/intellij.md [hunk 2]
```diff
@@ -11,16 +11,32 @@ title: IntelliJ IDEA
...
+
+<img width="461" alt="Gitpod browser extension configuration" src="/images/editors/inteliJ.gif">
+
 ## Browser (via Jetbrains Projector)
 
 Did you know you can run JetBrains IDEs in a browser? Thanks to [JetBrains Projector](https://lp.jetbrains.com/projector/) IntelliJ-based and Swing-based IDEs can run on servers, allowing you to access them from anywhere using your browser. Here's an example template: [gitpod-io/template-jetbrains-intellij-idea](https://github.com/gitpod-io/template-jetbrains-intellij-idea).
 
 Simply click "Open in Gitpod" button to launch IntellIJ IDEA in your browser.
 
+### FAQs
```
Comment (akosyakov) [inline src/routes/docs/editors/intellij.md:32]: it should go before ##Browser

**Comment by akosyakov** [comment]
@filiptronicek could you upgrade phpstorm as well

File: src/routes/docs/editors/goland.md (hunk 1 above)
Comment (filiptronicek) [reply src/routes/docs/editors/goland.md:22]: Agreed, a bit unimportant in this context.

File: src/routes/docs/editors/pycharm.md [hunk 3]
```diff
@@ -11,15 +11,31 @@ title: PyCharm
...
+
+**I changed my IDE preference, but Gitpod still opens VS Code, what should I do?** You must start a new workspace to see the JetBrains IDE prompt. Workspaces created before the preferences are updated will not show the JetBrains prompt. Workspaces will respect user preferences on start-up. _If you change your preferences you must stop your workspace and restart it_. Also, do note that when a workspace restarts, the Code With Me link will also update/change.
+
+**Do you need a license to use the JetBrains integration?** [Licensing](https://www.jetbrains.com/help/idea/remote-development-starting-page.html#licensing) is required from the desktop client.
+
 ## Browser (via Jetbrains Projector)
 
-Did you know you can run JetBrains IDEs in a browser? Thanks to [JetBrains Projector](https://lp.jetbrains.com/projector/) IntelliJ-based and Swing-based IDEs can run on servers, allowing you to access them from anywhere using your browser. Here's an example template: [gitpod-io/template-jetbrains-pycharm](https://github.com/gitpod-io/template-jetbrains-pycharm).
```
Comment (akosyakov) [inline src/routes/docs/editors/pycharm.md:20]: Should be revered

Comment (filiptronicek) [reply src/routes/docs/editors/pycharm.md:20]: Yikes, that's an oopsie that slid in.

**Pull request merged**